supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -21,3 +21,72 @@ When the format _day Month year_ causes a presentation or clarity issue, use ins
 
 [[product-names-versions-ref]]
 == Product names and version references
+
+////
+PROPOSED ADDITIONS?
+master-docinfo.xml specific and portal-specific version variables
+URL fragments and composed links?
+correct sequence when combining attributes
+ensure proper sequence for compound attributes
+syntax recommendations: what separators to use? dashes/hyphens, underscores?
+////
+
+Whenever you refer to the name of your product in full, or in its abbreviated form, or when you refer to the major and minor version of your product, avoid using hard-coded references and use attributes instead.
+
+Only use hard-coded version references if the version that you are referring to in that particular case never changes.  
+
+=== Attribute file
```

Comment:
@inoxx03 I thought that in mod docs there's a guideline about starting every module with a heading 1 and using offsets in the assembly to define the nesting.

Reply:
@inoxx03 You are absolutely correct! But ironically, the Supplementary Style Guide document is not (yet) modular.  It's still a monolith (I was surprised to find out about that after having cloned the repo). That being said, I'd be happy to lead by example and change my chapter into a properly chunked and formatted module. However we should all adopt that approach in our individual contributions and, going forward, also make it an action item to retroactively modularize the existing monolithic sections.

Reply:
@stoobie yeah for now, we didn't follow the mod docs layout because we ported what we had from the glossary of terms and conventions in here. I'm definitely open to tackling it at some point, but I didn't think it was worthwhile to worry about transforming all of that right now.